### ISSUE TITLE
Can not exit python with MxNet?

### ISSUE THREAD

**Issue by NMC-DAVE** [yajiedesign/mxnet#8]
Hi,
    I install the MxNet (build windows update 20170406, 20170406_mxnet_x64_vc14_cpu.7z and vc14 base package) on windows 7 64bit system and anaconda2 （python 2.7）。In python command， I can run "import mxnet as mx", but enter "exit()", the python stopped and can not exit. So, did I miss something?
very thanks!

**Comment by yajiedesign** [comment]
if you run "import mxnet" success,you not loss any thing.and exit() may is some bug.you can run xx.py. it will normal exit

**Comment by NMC-DAVE** [comment]
Hi yajie,
    I put the code into xx.py, and it still didn't work. The program output correctly, but stopped forever.

**Comment by ArtanisCV** [comment]
I also encountered this issue. I have tried with 20170125_mxnet_x64_vc12_cpu.7z (v0.9.3), and it was OK. After this release, vc12 version is no longer provided. However, with the vc14 version, python cannot exit successfully (will halt forever) if I import mxnet. It seems there is some bug.

**Comment by yajiedesign** [comment]
yes it is bug.No reason has been found yet. I can't reproduce the problem steadily.

**Comment by xiaogehh** [comment]
I have enconter this problem.
system:windows server 2008(64 bit)
python 2.7.9       anaconda 2.2.0
Using laterest version 20170905_mxnet_x64_vc14_cpu.7z,it also have this bug...

Even in C++，(import the libmxnet.dll and libmxnet.lib from this python version release) ,it also can't exit

**Comment by austingg** [comment]
I have install mxnet by pip, the problem also occurs. 
System: Windows 7
Python: 2.7.13 miniconda 
nightly build version

**Comment by austingg** [comment]
@yajiedesign  
here are more details:
```shell
python -dv hello-mxnet.py (import mxnet as mx)
```

```shell
# cleanup[2] stat
# cleanup[2] cv2.multicalib
# cleanup[2] numpy.lib.format
# cleanup[2] numpy.testing.nosetester
# cleanup[2] cryptography.hazmat.primitives.asymmetric.rsa
# cleanup[2] mxnet.executor_manager
# cleanup[2] mxnet.rnn.rnn_cell
# cleanup[2] cv2.ogl
# cleanup[2] time
# cleanup[2] idna
# cleanup sys
# cleanup __builtin__
# cleanup ints: 1718 unfreed ints
# cleanup floats: 287 unfreed floats

```

I also tried to build mxnet from source, and attach the python repl, when I type `exit()`,  VisualStudio debug window close, seems it believe the python process has exited.

**Comment by NMC-DAVE** [comment]
not resolve too.





At 2017-09-19 11:38:01, "Yubin Wang" <notifications@github.com> wrote:


I have install mxnet by pip, the problem also occurs.
System: Windows 7
Python: 2.7.13 miniconda
nightly build version
